iconservice/inner_call/__init__.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 ICON Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+from ..icon_constant import PREP_COUNT
+from ..iconscore.icon_score_context import IconScoreContext
+
+if TYPE_CHECKING:
+    from ..base.block import Block
+    from ..prep.data.prep import PRep
+    from ..prep.data.prep_container import PRepContainer
+
+
+def get_preps(context: IconScoreContext):
+    preps: 'PRepContainer' = context.engine.prep.preps
+    prep_result = []
+
+    for prep in preps:
+        p_rep: 'PRep' = context.engine.prep.preps.get(prep.address)
+        data = {
```

Comment:
remove "rep" and "irepUpdateBlockHeight".
check [protocol doc](https://docs.google.com/document/d/1WkK2foaxauqTTT-GRyOaabgjhcrPB5049Qt6GENpAnI/edit#heading=h.cemoeknli1c)